ga4gh/datamodel/ontologies.py

```diff
@@ -8,36 +8,45 @@
 import ga4gh.protocol as protocol
 
 
-class Ontology(object):
+class OntologyTermMap(object):
     """
-    At this moment an "Ontology" is just a map between names and IDs (e.g.
-    in Sequence Ontology we would have "SO:0001583 <-> missense_variant")
-    This is a tempotrary solution and must be replaced by more comprehensive
-    ontology object.
+    A bidectional map between ontology names and IDs (e.g. in Sequence
+    Ontology we would have "SO:0001583 <-> missense_variant"). This
+    implementation uses a tab separated TXT file: "id\tname"
 
-    This implementation uses a tab separated TXT file: "id\tname"
+    This is a temporary solution and must be replaced by more comprehensive
+    ontology object.
     """
     def __init__(self, localId):
         self._localId = localId
         self._dataUrl = None
+        self._sourceName = None
         self._nameIdMap = dict()
         self._idNameMap = dict()
 
     def populateFromFile(self, dataUrl):
         """
         Populates this ontology map from the specified dataUrl.
+        This reads the ontology term name and ID pairs from the
+        specified file.
         """
         self._dataUrl = dataUrl
-        self.readOntology(self._dataUrl)
+        # TODO shouldn't sourceName be the name of the ontology, (e.g,
+        # sequence_ontology?)
```

Comment:
+100 on moving forward with this ontology map changes.  There
will be much more work with ontologies in the future.

Jerome Kelleher <notifications@github.com> writes:
> In ga4gh/datamodel/ontologies.py:
> 
> >          """
> >          self._dataUrl = dataUrl
> > -        self.readOntology(self._dataUrl)
> > +        # TODO shouldn't sourceName be the name of the ontology, (e.g,
> > +        # sequence_ontology?)
> 
> Updated to remove the comments and make the sourceName == name. We definitely
> need the think how we handle ontologies through again, and allowing the user
> specify the map used is a good start. I think we should do this after merging
> into master though.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>